packages/plugin-drizzle/src/storage.ts

```diff
@@ -93,18 +117,21 @@ export async function registerTriggers<
   tables: string[],
   endCursor: Cursor,
   idColumn: string,
+  indexerId: string,
 ) {
   try {
     for (const table of tables) {
       await tx.execute(
-        sql.raw(`DROP TRIGGER IF EXISTS ${table}_reorg ON ${table};`),
+        sql.raw(
+          `DROP TRIGGER IF EXISTS ${table}_reorg_${indexerId} ON ${table};`,
```

Comment:
We should add an helper function called `triggerName(table, indexerId)` to generate the name.

Reply:
Then use the helper function everywhere we need the trigger name.